### PR TITLE
Improve reverse_readlines performance

### PR DESCRIPTION
**Pull request by JSCU-CNI**
This PR improves `year_rollover_helper` performance by setting the `reverse_readlines` `chunk_size` parameter to 8 MB. This change also makes sure the function does not break when encountering a log line without a valid `re_ts`.

### CODE REVIEW

**Review by Schamper — CHANGES_REQUESTED** [review]

**Comment by JSCU-CNI** [comment]
Thanks for the suggestions! Implemented in [fa4c04d](https://github.com/fox-it/dissect.target/pull/204/commits/fa4c04d633abe17cbcd3ccb682759f3b70e186f3).

File: dissect/target/helpers/utils.py [hunk 1]
```diff
@@ -82,12 +88,18 @@ def year_rollover_helper(
...
             if not line:
                 continue
 
-            timestamp = re.search(re_ts, line).group(0)
-            relative_ts = datetime.strptime(timestamp, ts_format)
+            timestamp = re.search(re_ts, line)
+
+            if not timestamp:
+                log.warning(f"No timestamp found in one of the lines in {path}!")
+                log.debug(f"Skipping this line: {line}")
```
Comment (Schamper) [inline dissect/target/helpers/utils.py:99]:
```suggestion
                if not warned:
                    log.warning("No timestamp found in one of the lines in %s!", path)
                    warned = True
                log.debug("Skipping line: %s", line)
```

File: dissect/target/helpers/utils.py [hunk 2]
```diff
@@ -82,12 +88,18 @@ def year_rollover_helper(
 
     with fsutil.open_decompress(path, "rt") as fh:
```
Comment (Schamper) [inline dissect/target/helpers/utils.py:89]:
```suggestion
    with fsutil.open_decompress(path, "rt") as fh:
        warned = False
```

**Review by Schamper — CHANGES_REQUESTED** [review]
Fixed log formatting + a small suggestion to prevent flooding of warnings

**Comment by JSCU-CNI** [comment]
Thanks, implemented in [68f4672](https://github.com/fox-it/dissect.target/pull/204/commits/68f467206fb983b77a5d64d6fdcc87497756344c).

**Review by Schamper — APPROVED** [review]

**Pull request merged**